datatypes/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+pub mod persistence;
```

Comment:
These are here as they are used by regionalextracts and the backend?

Reply:
Yes, these are shared data types between the regionalextracts tool and the backend, so I extracted it into a seperate crate